week-14/practices/05-Navigation/README.md

```diff
@@ -1,22 +1,153 @@
 # Practice: Navigation
 
-In this practice you will learn how to create navigation for your Route
-paths using `Link` and `NavLink`.
+In this practice you will learn how to create navigation for your Route paths
+using `Link` and `NavLink`.
 
 ## Phase 0: Setup
 
-Download the [starter][starter]. `cd` into the folder and run the `npm install`
-command in your terminal.
+Download the [starter][starter]. `cd` into the folder and run the `npm install`command in your terminal.
 
 Next run `npm start` and check your browser to see if your code is running
 properly.
 
 Check your browser. You should only see an `h1` saying App Components.
 
-Now take a look at the code. In your `App.js` you will see the already
-created routes for each component.
+Now take a look at the code. In your `App.js` you will see the already created
+routes for each component. You now want to create clickable Links for each of
+those routes.
 
 ## Phase 1: Link Navigation
 
-Import the Link component from `react-router-dom`. You will use this
-component instead of the `<a>` element to which you are accustomed.
+In your App.js file, import the Link component from `react-router-dom`. You will
+use this component instead of the `<a>` element to which you are accustomed.
+
+Beneath the `h1` create a `nav` element with 2 classes: `comp` and `nav`. Inside
+the `nav` element add an unordered list element then a list-item element inside
+the unordered list element. Now use your Link component to create a clickable
+like to the path for Home. The child text inside the Link component should say
```

Comment:
**Line 27:** Did you mean "link" instead of "like"?

Reply:
edited